src/testdrive/src/action.rs

```diff
@@ -76,6 +76,8 @@ pub struct Config {
     pub reset_materialized: bool,
     /// Emit Buildkite-specific markup.
     pub ci_output: bool,
+    /// A random number to distinguish each TestDrive run.
+    pub seed: Option<u32>,
```

Comment:
nonce is probably a better name since seed suggests it'd be used to create an rng, whereas nonce suggests it's a random num to be used as-is

Reply:
I forgot that we expose this parameter to tests as `${testdrive.seed}`, so please ignore this comment - better to maintain consistency.

Reply:
I agree that `nonce` would have been the better name from the beginning. 😅 

I could change it in the CLI but all of the testdrive test already use `seed`. If you grep for `testdrive.seed` you'll see there are quite some instances. If the CLI calls it `nonce` and the rest still uses `seed` that could be confusing.

I would either change it everywhere or leave it as `seed` in the CLI as well, WDYT?

Reply:
Ahh, I wrote this without refreshing first ...